demo/src/main/res/raw/dark_style.css

```diff
@@ -0,0 +1,18 @@
+body {
+    font-family: sans-serif;
+    font-size: 14;
+    background-color: #000000;
+    color: #FFFFFF;
```

Comment:
Ideally, we should inject the background and content colors from the theme :D But we'll can see that later ;)

Reply:
By the way, I see that in Innovation cup theme the links in the changelog are Orange. Maybe we can create an issue to manage that later.

Reply:
Yes, we can fix the bug and inject theme colors at the same time in another PR :+1: